crates/gen/src/types/bstr.rs

```diff
@@ -24,7 +24,7 @@ pub fn gen_bstr() -> TokenStream {
                     return 0;
                 }
 
-                unsafe { SysStringLen(self) as usize }
+                unsafe { super::System::OleAutomation::SysStringLen(self) as usize }
```

Comment:
This seems wrong. Created an issue: https://github.com/microsoft/win32metadata/issues/517